.github/workflows/internal-build.yml

```diff
@@ -166,10 +166,21 @@ jobs:
   #     SLACK_WEBHOOK_URL: '${{ secrets.SLACK_WEBHOOK }}'
 
   sonar-check:
-    uses: tw-mosip/kattu/.github/workflows/npm-sonar-analysis.yml@inji-add-release-keystore-step-for-sonar
-    with:
-      SERVICE_LOCATION: '.'
-    secrets:
-      SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-      ORG_KEY: ${{ secrets.ORG_KEY }}
-      SLACK_WEBHOOK_URL: '${{ secrets.SLACK_WEBHOOK }}'
\ No newline at end of file
+    runs-on: ubuntu-latest
+    steps:
+    - name: Generate keystore
```

Comment:
Can we not add this step in kattu? 
Is there any discussion happened?

Reply:
Removed building apk as of now to test if minimal steps pass